src/test/java/uk/gov/ons/ctp/response/action/export/service/NotificationFileCreatorTest.java

```diff
@@ -30,7 +30,7 @@
 public class NotificationFileCreatorTest {
 
   private static final SimpleDateFormat FILENAME_DATE_FORMAT =
-      new SimpleDateFormat("ddMMyyyy_HHmm");
+      new SimpleDateFormat("ddMMyyyy_HHmmss");
```

Comment:
The format in the spec is ISO 8601 with dashes instead of colons (YYYY-MM-DDTHH-MM-SS)

Reply:
Fixed.